Editor/TextProvider.cs

```diff
@@ -182,6 +182,23 @@ internal sealed class TextProvider
             { Strings.TabAnywhere, "GameLift Anywhere" },
             { Strings.TabEC2, "Managed EC2" },
             { Strings.TabHelp, "Learning Resources" },
+            
+            { Strings.LabelHelpTitle, "Help & Documentation" },
+            { Strings.LabelHelpTitleDescription, "Learn how to integrate and deploy games for hosting on GameLift. The GameLift service manages game server deployment, operation, and scaling. You can deploy custom-built game servers or use GameLift Realtime Servers to configure lightweight game servers for your game." },
+            { Strings.LabelHelpReportIssues, "Report Issues \uf086" },
```

Comment:
What is `\uf086`? Is this intentional? Or a result of a copy paste of a string from someplace else?

Reply:
It is a unicode character for the external link icon. This is provided by the Font Awesome font.